nextflow/modules/contig_search_taxonomy/process/diamond.nf

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+
+process DIAMOND {
+  label 'process_high'
+  label 'process_long'
+  
+  tag "diamond blastp on ${orfs.simpleName}"
```

Comment:
I think it will change if we overhaul how we do names. 
For consistency with existing pipelines I would suggest we implement something like https://nf-co.re/mag/usage#tsv-input-file